drivers/serial/Kconfig.ns16550

```diff
@@ -72,6 +72,15 @@ config UART_NS16550_SIMULT_ACCESS
 	  When enabled, NS16550 supports IO, MMIO, PCIe UART devices simultaneously.
 	  For io-mapped instances, io-mapped DTS property need to be added in dtsi.
 
+config UART_NS16550_PARENT_INIT_LEVEL
+	bool "Boot level based on parent node"
```

Comment:
Would it make sense to add `depends on ACPI` here?

Reply:
Or perhaps `depends on ACPI && PCIE`?

Reply:
better default y if ACPI, right?

Reply:
> better default y if ACPI, right?

I guess that makes sense. And you can then drop the board-specific setting for RPL.

Reply:
updated with default y if  ACPI enabled. Please review and approve if no more comments